src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -605,41 +606,51 @@ export function getFunnelTableVisualDef(visualId: string, viewName: string, even
       Width: '120px',
     });
 
-    const fieldIdRate = uuidv4();
     if (index === 0) {
-      groupBy.push({
-        NumericalDimensionField: {
-          FieldId: fieldIdRate,
-          Column: {
-            DataSetIdentifier: viewName,
-            ColumnName: 'rate',
-          },
+      continue;
```

Comment:
why excluding the first event here?

Reply:
first event don't have conversation rate, we just need a 'event_name' column.